kong/plugins/aws-lambda/schema.lua

```diff
@@ -20,11 +22,13 @@ return {
     },
     aws_key = {
       type = "string",
-      required = true,
     },
     aws_secret = {
       type = "string",
-      required = true,
+    },
+    use_ec2_iam_role = {
```

Comment:
Our policy is that no plugin config already in the database should be left in a previous version of the plugin's schema. This is because we want to avoid complicated code paths handling different versions of the same plugin's schema in the plugin business logic itself. This is a policy that has unfortunately not always been followed, and such oversight lead us to a few bugs/overhead maintenance issues recently.

So: new schema value, new migration for Cassandra/PostgreSQL :)